.github/workflows/release.yaml

```
@@ -0,0 +1,91 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      target-branch:
+        description: "Branch to release"
+        required: true
+        type: choice
+        default: 'main'
+        options:
+        - main
+      version:
+        description: "Version to release (e.g. 1.1.0 or 1.2.0.alpha1)"
+        required: true
+        type: string
+      dry-run:
+        description: "Dry run (Skips remote operations)"
+        required: true
+        type: boolean
+        default: false
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Validate Workflow branch
+        run: |
+          BRANCH_NAME=${GITHUB_REF##*/}
+          if [[ "main" != "${BRANCH_NAME}" ]]; then
+            echo "ERROR: Release Workflow must be dispatched on 'main' branch."
+            exit 1
+          fi
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: refs/heads/${{ inputs.target-branch }}
+          fetch-depth: 0
+
+      - name: Validate version
+        run: |
+          VERSION_REGEX='^[0-9]+\.[0-9]+\.[0-9]+(\.(alpha|beta|rc)[0-9]+)?$'
+          if [[ ! "${{ inputs.version }}" =~ $VERSION_REGEX ]]; then
+            echo "ERROR: Invalid version specified: '${{ inputs.version }}'. Please enter a valid SemVer version, like '1.2.3' or '1.0.0.alpha1'."
+            exit 1
+          fi
+          if git rev-parse -q --verify "refs/tags/${{ inputs.version }}" >/dev/null; then
```

```suggestion
          if git rev-parse -q --verify "refs/tags/${{ inputs.version }}" > /dev/null; then
```
Increases readability (don't know if it breaks something)

---

done